.ci_support/migrations/python313.yaml

```diff
@@ -0,0 +1,42 @@
+migrator_ts: 1724712607
+__migrator:
+    commit_message: Rebuild for python 3.13
```

Comment:
Have pulled in the Python 3.13 migrator from PR ( https://github.com/conda-forge/numcodecs-feedstock/pull/109 ) into this PR

As a result, this also enables Python 3.13 builds with this release

This aligns with equivalent changes upstream to support Python 3.13:

* https://github.com/zarr-developers/numcodecs/pull/576
* https://github.com/zarr-developers/numcodecs/pull/591